### PR TITLE
retrieve access token only by default

### PR DESCRIPTION
**Pull request by xcoulon**
Signed-off-by: Xavier Coulon <xcoulon@redhat.com>

### CODE REVIEW

File: README.adoc [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+make install
+```
+
+== How to use it
+
+```
+osio login --help
+```
+
+On macOS, you can register a password for a service/account pair which can be used with the `osio` command so you don't need to input your OpenShift.io account's password each time:
```
Comment (chmouel) [inline README.adoc]: you may want to add that Linux *should* be supported following this instructions : 

https://github.com/zalando/go-keyring#linux

File: cmd/login.go [hunk 2]
```diff
@@ -107,8 +112,12 @@ func login(cmd *cobra.Command, args []string) {
 		if err != nil {
 			log.Fatalf("failed to parse the tokens retrieved upon login: %v", err)
 		}
-		fmt.Printf("access token: %s\n\n", tokens.AccessToken)
-		fmt.Printf("refresh token: %s\n\n", tokens.RefreshToken)
+		if accessTokenOnly {
+			fmt.Printf(tokens.AccessToken)
```
Comment (chmouel) [inline cmd/login.go:116]: 👍

thanks,

File: README.adoc (hunk 1 above)
Comment (xcoulon) [reply README.adoc]: done in 066e2fe

**Comment by chmouel** [comment]
thanks for addressing this, merged!

**Comment by xcoulon** [comment]
great, thanks for your feedback @chmouel!

**Pull request merged**